eventservice/api/controllers/events.py

```diff
@@ -681,6 +647,34 @@ def images_delete(event_id, image_id):
     return success_response(202, msg, str(event_id))
 
 
+def group_event_admin_operation_permission_check(req_data, event_id=None):
+    if req_data.get('createdByGroupId'):
+        group_memberships = list()
+        try:
+            _, group_memberships = get_group_memberships()
+        except Exception as ex:
+            __logger.exception(msgs.ERR_MSG_GET_GROUP_MEMBERSHIP)
+            abort(500)
+        event = None
+        if event_id:
+            try:
+                db = get_db()
+                event = db['events'].find_one({'_id': ObjectId(event_id)}, {'_id': 0})
+            except Exception as ex:
+                __logger.exception(msgs.ERR_MSG_GET_EVENT)
+                abort(500)
+
```

Comment:
```suggestion

            # Event not found
            if not event:
                abort(404)

```

Similarly here as well. Let's return HTTP 404 if the event is not found.